presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -82,6 +82,10 @@ private static ConnectionFactory connectionFactory(BaseJdbcConfig config, MySqlC
         connectionProperties.setProperty("useUnicode", "true");
         connectionProperties.setProperty("characterEncoding", "utf8");
         connectionProperties.setProperty("tinyInt1isBit", "false");
+        config.setListSchemasIgnoredSchemas(ImmutableSet.<String>builder()
+                .addAll(config.getIgnoredSystemSchemas())
+                .addAll(mySqlConfig.getAdditionalIgnoredSchemas())
+                .build());
```

Comment:
We still need to let the user override this value entirely. It's possible they may want to actually see information_schema or the other config. We should modify the config defaults inside of the Connector's module binding. For example, inside of MysqlClientModule add the line:

```java
configBinder(binder).bindConfigDefaults(BaseJdbcConfig.class, baseJdbcConfig -> {
            baseJdbcConfig.setXXX
        });
```

You should check all the other JDBC modules which rely on `BaseJdbcConfig` and see if they need this update